_layouts/index.html

```diff
@@ -27,9 +29,11 @@ <h2 id="question">{{ t.pagecontent.question }}</h2>
           <p>{{ t.pagecontent.what }}</p>
         </div>
         <div class="col-2">
+          <div class="highlight-wrapper" tabindex="0">
```

Comment:
These don't need `tabindex` (and I'd argue they are actively not desirable to tab between).

Reply:
So how do you scroll them without mouse if overflow is set to auto?

Reply:
I can't see any situation in which you'd need to scroll them or be able to resize the page such that they scroll.